src/core/Do.Architecture/Architecture/LayerBase.cs

```diff
@@ -4,6 +4,8 @@ public abstract class LayerBase : ILayer
 {
     protected ApplicationContext Context { get; private set; } = default!;
 
+    public string Id => this.GetType().FullName ?? this.GetType().Name;
```

Comment:
just `Name` is sufficient, no need to include namespace of layer class.

Reply:
this is default behaviour, and two layers having the same name is really unlikely. if a conflict occurs, you can always override